### ISSUE TITLE
Populate proper TRICARE taxonomy for Lovell Conent complete

### ISSUE THREAD

**Issue by Dottisea** [department-of-veterans-affairs/va.gov-cms#12322]

**Comment by davidmpickett** [comment]
Notes from Taxonomy meeting today. 

- Justin Warren and Blake Scates from VHA DM to communicate decisions to Lovell
- Steve Tokar from VHA DM will lead drafting new content for the taxonomy
- Sitewide Content team will review and approve content
- @davidconlon said to Danielle Thierry something to the effect of "We should talk about the workflow of updating taxonomy terms in Drupal."

**Comment by Dottisea** [comment]
Dependency: Waiting on content from content team before this can be done

**Comment by davidmpickett** [comment]
Pulling this thread out from the email chain:

For all the new terms being added to the taxonomy (light blue in attached excel), the Facilities team will need to specify a Health Service API ID. This is a required field.

From @davidconlon - This is applicable to Lighthouse only and there is a rubric which we use for the others.

[Updated Lovell TRICARE Description texts_2_st.xlsx](https://github.com/department-of-veterans-affairs/va.gov-cms/files/10548051/Updated.Lovell.TRICARE.Description.texts_2_st.xlsx)

**Comment by davidconlon** [comment]
Content team will populate the taxonomy with the Tricare content.

**Comment by kmariepat-cityfriends** [comment]
The work has been done, Dave C. and Jenny HL are now reviewing

**Comment by davidmpickett** [comment]
I added some Acceptance Criteria so we can be clearer on when this is closeable

**Comment by davidmpickett** [comment]
Based on @davidconlon's experience this morning, we should probably create a KB article that explains the process of requesting a Drupal admin to create the Terms because of the Health Service API. @omahane

**Comment by omahane** [comment]
@kmariepat-cityfriends Do we know if Lovell editors have been informed about this?

**Comment by kmariepat-cityfriends** [comment]
@omahane I don't think so, I have a sync today with VHA Digital Media and I understand that group to be the people that own communicating to editors. I will report back on this card today after bringing this up in that sync

**Comment by kmariepat-cityfriends** [comment]
@omahane VHA digital media is going to communicate this to editors this week. In next week's Lovell office hours we will ask editors if they have any questions or issues with the service taxonomy

**Comment by omahane** [comment]
@kmariepat-cityfriends Do you know if this communication has happened?

**Comment by kmariepat-cityfriends** [comment]
@omahane assuming you've seen my above comment, I can ask Blake and Justin at VHA Digital Media to explicitly confirm they've let editors know

**Comment by kmariepat-cityfriends** [comment]
@omahane "Justin informed the Lovell editors on 3/28"
